core/storage/user/gae_models.py

```diff
@@ -672,3 +672,119 @@ def construct_model_id(cls, user_id, skill_id):
             str. The model id corresponding to the given user and skill.
         """
         return '%s.%s' % (user_id, skill_id)
+
+
+class UserContributionScoringModel(base_models.BaseModel):
```

Comment:
Are all of these methods being tested? If not, can you add tests for those which are untested?

Reply:
Oops had missed some of them. Added tests for all of them. Thanks!